tests/php/test-class-amp-theme-support.php

```diff
@@ -1906,7 +1906,7 @@ public function test_prepare_response_standard_mode_non_amp( $with_amp_live_list
 
 		$sanitized_html = AMP_Theme_Support::prepare_response( $original_html, [ ConfigurationArgument::ENABLE_OPTIMIZER => false ] );
 
-		$this->assertStringContainsString( '<html>', $sanitized_html, 'The AMP attribute is removed from the HTML element' );
+		$this->assertStringContainsString( '<html ', $sanitized_html, 'The AMP attribute is removed from the HTML element' );
```

Comment:
This isn't testing that the `amp` attribute was removed. I suggest something like this instead:

```suggestion
		$this->assertDoesNotMatchRegularExpression( '/<html[^>]*amp[^>]*>/', $sanitized_html, 'The AMP attribute is removed from the HTML element' );
```

Reply:
I have updated the test case.